### PR TITLE
Correct top bar offsets and positions on Android + correct blur as a bonus ✨

### LINKED ISSUE #16141: Chat header is not rendered properly
## Bug Report

The chat header is rendered behind the status bar in Android and when the keyboard is open, the header is not rendered correctly.

cc: @alwx 

<table>
  <tr>
    <th>
    Empty chat screen w/o keyboard opened
    </th>
    <th>
     Empty chat screen w/ keyboard opened
    </th>
    <th>
     Chat screen with one message
    </th>
  </tr>
  <tr>
    <td>
      
    </td>
    <td>
      
    </td>
 <td>
      
    </td>
  </tr>
</table>

### Expected behaviour

The chat header should not render behind the status bar

### Reproduction

1) Open Status
2) Navigate to any chat

### Additional Information

- Status Version: nightly [02/06]
- Operating System: Android

### PR DESCRIPTION
**Pull request by alwx**
Fixes #16141, also fixes the blur (makes it the same as the blur for composer)

### Summary

Updates the height calculation for Android.

Screenshot_20230602-124844

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

status: ready

### CODE REVIEW

File: src/status_im2/contexts/chat/messages/navigation/view.cljs [hunk 1]
```diff
@@ -51,13 +52,13 @@
                                                           :extrapolateRight "clamp"})]
     [rn/view {:style style/navigation-view}
      [reanimated/blur-view
-      {:blurAmount   32
-       :blurType     (colors/theme-colors :xlight :dark)
-       :overlayColor :transparent
-       :style        (style/animated-blur-view all-loaded? opacity-animation status-bar-height)}]
+      {:blur-amount 16
```
Comment (OmarBasem) [inline src/status_im2/contexts/chat/messages/navigation/view.cljs]: I think the top bar blur should be matching with the composer's blur 
`:blur-amount 20`

**Comment by OmarBasem** [comment]
Top bar blur does not look right. In the designs, the blur of the top bar and the composer looks matching. Currently the top bar looks too transparent.

**Comment by alwx** [comment]
Just fixed the styling — they are now the same (see screenshot above)

**Comment by alwx** [comment]
Some extra styling (like # before the channel name), paddings, etc will be done later on when working on chat UI (and here: https://github.com/status-im/status-mobile/issues/16089) — this one is mainly to fix the offset calculations.

**Comment by OmarBasem** [comment]
On Android, after the top bar animation completes, the whole screen gets a layer of opacity. Also, scrolling gets locked for a few seconds. That issue wasn't there before

**Comment by alwx** [comment]
@OmarBasem fixed in the recent commit!

**Comment by alwx** [comment]
@OmarBasem please review

**Review by OmarBasem — APPROVED** [review]

**Comment by alwx** [comment]
Partially fixed here: https://github.com/status-im/status-mobile/pull/16148

**Review by ilmotta — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/navigation/style.cljs [hunk 2]
```diff
@@ -17,33 +16,47 @@
...
+   :height           navigation-bar-height
+   :background-color (colors/theme-colors colors/white-opa-70 :transparent)
+   :display          :flex
+   :flex-direction   :row
+   :overflow         :hidden})
+
+(defn animated-background-view
+  [enabled? animation]
+  (reanimated/apply-animations-to-style
+   (when enabled?
```
Comment (ilmotta) [inline src/status_im2/contexts/chat/messages/navigation/style.cljs:33]: Same as in another similar PR, we could probably surround the whole `apply-animations-to-style` call in a `when`.

**Comment by alwx** [comment]
@status-im/mobile-qa any chance to get this one reviewed? people say it's blocking

**Comment by pavloburykh** [comment]
@alwx will be able to take into work tomorrow morning if noone picks it up sooner

**Comment by qoqobolo** [comment]
### ISSUE 1: Wrong position of the header on iPhone SE when opening an empty 1-1 chat


https://github.com/status-im/status-mobile/assets/67952253/82878567-53f4-4582-aaa0-49224166c30c

**Comment by qoqobolo** [comment]
Thanks for the fix @alwx!
Let's merge this PR to unblock development, I'll log other iOS related issues separately.

**Pull request merged**